doc/mkdocs/docs/api/basic_json/contains.md

```diff
@@ -39,7 +47,8 @@ Strong exception safety: if an exception occurs, the original value stays intact
 ## Exceptions
 
 1. The function does not throw exceptions.
-2. The function can throw the following exceptions:
+2. See 1.
```

Comment:
Nitpicky, but I'd rather repeat `Logarithmic in the size of the container.` here. The level of indirection for that short sentence confuses me.